packages/ui-components/src/components/toolbar.tsx

```diff
@@ -966,9 +966,11 @@ class ToolbarPopupOpener extends ToolbarButton {
       icon: ellipsesIcon,
       onClick: () => {
         this.handleClick();
-      }
+      },
+      tooltip: 'More commands'
```

Comment:
Needs translations (and labels need it too).

Reply:
Thanks @krassowski